### PR TITLE
add enable config for batch write

### PR DESCRIPTION
**Pull request by marsishandsome**
default disable batch write

### CODE REVIEW

**Comment by marsishandsome** [comment]
@zhexuany @birdstorm PTAL

File: core/src/main/scala/com/pingcap/tispark/TiConfigConst.scala [hunk 1]
```diff
@@ -39,6 +39,8 @@ object TiConfigConst {
   val CACHE_EXPIRE_AFTER_ACCESS: String = "spark.tispark.statistics.expire_after_access"
   val SHOW_ROWID: String = "spark.tispark.show_rowid"
   val DB_PREFIX: String = "spark.tispark.db_prefix"
+  val WRITE_ALLOW_SPARK_SQL: String = "spark.tispark.write.allow_spark_sql"
```
Comment (zhexuany) [inline core/src/main/scala/com/pingcap/tispark/TiConfigConst.scala]: add a todo? We need to remove this flag once batch write is stable.

**Pull request merged**